content/en/opentelemetry/otel_collector_datadog_exporter.md

```diff
@@ -662,12 +666,34 @@ The OpenTelemetry Collector has [two primary deployment methods][20]: Agent and
 
 Datadog provides out-of-the-box dashboards that you can copy and customize. To use Datadog's out-of-the-box OpenTelemetry dashboards:
 
-1. Install the [OpenTelemetry integration][23].
+1. Install the [OpenTelemetry integration][24].
 2. Go to **Dashboards** > **Dashboards list** and search for `opentelemetry`:
 
    {{< img src="metrics/otel/dashboard.png" alt="The Dashboards list, showing two OpenTelemetry out-of-the-box dashboards: Host Metrics and Collector Metrics." style="width:80%;">}}
 
-The **Host Metrics** dashboard is for data collected from the [host metrics receiver][21]. The **Collector Metrics** dashboard is for any other types of metrics collected, depending on which [metrics receiver][22] you choose to enable.
+The **Host Metrics** dashboard is for data collected from the [host metrics receiver][25]. The **Collector Metrics** dashboard is for any other types of metrics collected, depending on which [metrics receiver][26] you choose to enable.
+
+
+### Containers Overview dashboard
+
+Container metrics in Datadog are generated by the [dockerstats][27] receiver. When used in conjunction with the Datadog Exporter, it translates them to their Datadog counterparts. Besides the default attributes, some additional ones need to be enabled. To do so, use the following configuration:
```

Comment:
```suggestion
The [Docker Stats receiver][27] generates container metrics in Datadog. When used in conjunction with the Datadog Exporter, the receiver translates container metrics to their Datadog counterparts. 

Use the following configuration to enable additional attributes on the Docker Stats receiver that populate the containers overview dashboard:
```

Reply:
This isn't correct. The Docker Stats receiver doesn't generate metrics for Datadog. It generates them for the OpenTelemetry Collector, which Datadog then uses. The first sentence would be a bit incorrect here.

Reply:
Also, "the exporter" translates container metrics to their Datadog counterparts. Which even more underlines how bad it was for me to use the word "it" there and how it can be misunderstood :)

Reply:
Thanks for the clarification. I'm still having a bit of trouble understanding the various components, so please feel free to correct my writing. What do you think about this wording?

> The Docker Stats receiver generates container metrics for the OpenTelemetry Collector, which sends the metrics to Datadog. The Datadog Exporter then translates container metrics to their Datadog counterparts.
>
> Use the following configuration to enable additional attributes on the Docker Stats receiver that populate the containers overview dashboard:

Reply:
```suggestion
The [Docker Stats][27] receiver generates container metrics for the OpenTelemetry Collector. The Datadog Exporter translates container metrics to their Datadog counterparts.

Use the following configuration to enable additional attributes on the Docker Stats receiver that populate the containers overview dashboard:
```